modules/gitops-installing-argocd-cli-on-linux-using-rpm.adoc

```diff
@@ -0,0 +1,137 @@
+// Module is included in the following assemblies:
+//
+// * installing_gitops/installing-argocd-gitops-cli.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-installing-argocd-cli-on-linux-using-rpm"]
+= Installing the {gitops-title} CLI on Linux using an RPM
+
+For {op-system-base-full} version 8 or later, you can install the {gitops-shortname} `argocd` CLI as an RPM by using a package manager, such as `yum` or `dnf`. This allows the {gitops-shortname} `argocd` CLI version to be automatically managed by the system. For example, using a command like `dnf upgrade` upgrades all packages, including `argocd`, if a new version is available.
+
+.Prerequisites
+
+* You have an active {OCP} subscription on your Red Hat account.
+* You have root or `sudo` privileges on your local system.
+
+.Procedure
+
+. Register with Red Hat Subscription Manager:
++
+[source,terminal]
+----
+# subscription-manager register
+----
+
+. Pull the latest subscription data:
++
+[source,terminal]
+----
+# subscription-manager refresh
+----
+
+. List the available subscriptions:
++
+[source,terminal]
+----
+# subscription-manager list --available --matches '*gitops*'
+----
+
+. In the output for the previous command, find the pool ID for your {OCP} subscription and attach the subscription to the registered system:
++
+[source,terminal]
+----
+# subscription-manager attach --pool=<pool_id>
+----
+
+. Enable the repositories required by {gitops-title}:
++
+* For RHEL 8
+** Linux (x86_64, amd64)
++
+[source,terminal]
+----
+# subscription-manager repos --enable="gitops-1.12-for-rhel-8-x86_64-rpms"
+----
++
+** Linux on {ibmzProductName} and {linuxoneProductName} (s390x)
++
+[source,terminal]
+----
+# subscription-manager repos --enable="gitops-1.12-for-rhel-8-s390x-rpms"
+----
++
+** Linux on {ibmpowerProductName} (ppc64le)
++
+[source,terminal]
+----
+# subscription-manager repos --enable="gitops-1.12-for-rhel-8-ppc64le-rpms"
+----
++
+** Linux on ARM (aarch64, arm64)
++
+[source,terminal]
+----
+# subscription-manager repos --enable="gitops-1.12-for-rhel-8-aarch64-rpms"
+----
++
+* For RHEL 9
```

Comment:
nit: Do we need this documented separately for RHEL 9? I see the only change in the commands is OS version

Reply:
@varshab1210 next release, we will have only RHEL 9 and remove RHEL 8. so retaining this as is for now.